Updating images (#255)
OfficeDev/office-scripts-docs: docs/develop/scripting-fundamentals.md

Running this script creates the following data in the current worksheet:

> :::image type="content" source="../images/range-sample.png" alt-text="A sales record showing value rows, a formula column, and formatted headers.":::

Reviewer: ```suggestion
:::image type="content" source="../images/range-sample.png" alt-text="A  worksheet containing a sales record consisting of value rows, a formula column, and formatted headers.":::
```

> :::image type="content" source="../images/range-sample.png" alt-text="A  worksheet containing a sales record consisting of value rows, a formula column, and formatted headers.":::

Reviewer: ```suggestion
:::image type="content" source="../images/range-sample.png" alt-text="A worksheet containing a sales record consisting of value rows, a formula column, and formatted headers.":::
```

Running this script on the worksheet with the previous data creates the following table:

> :::image type="content" source="../images/table-sample.png" alt-text="A table made from the previous sales record.":::

Reviewer: ```suggestion
:::image type="content" source="../images/table-sample.png" alt-text="A worksheet containing a table made from the previous sales record.":::
```
